platform/platform-scheduler/src/test/java/ddf/platform/scheduler/CommandJobTest.java

```diff
@@ -85,50 +85,35 @@ public void testNullCommand() throws Exception {
     // given
     String command = null;
 
-    FirstArgumentAnswer captureInput = new FirstArgumentAnswer();
-
-    Session session = getSession(captureInput);
-
-    when(sessionFactory.create(notNull(InputStream.class), any(), any())).thenReturn(session);
-
     CommandJob job = getCommandJob();
 
     // when
     job.execute(getJobExecutionContext(command));
 
     // then
-    verifySessionCallsAndSessionClosed(command, session, 0);
+    /* we should not have a problem */
```

Comment:
❓ Isn't there anything we could verify to prove that we got here and things worked as expected, like we used to do?

Reply:
I couldn't find a good way. We were sort of artificially spying on private methods in `CommandJob` before by overriding methods, and then asserting that the code in that we put in was never called. I updated `CommandJobTest` to do a better job at only mocking what we want to test, and I added descriptions for all of the test methods.